src/common/utils/is-node-environment/index.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * @see https://github.com/jsdom/jsdom/releases/tag/12.0.0
+ * @see https://github.com/jsdom/jsdom/issues/1537
+ */
+const isJsDomEnvironment = () =>
```

Comment:
I feel it might be better to just check if we're rendering in node, rather than checking for jsdom. Some users might be rendering without jsdom which would fail this check.

Reply:
I agree it's quite jsdom centric. I think we still need the jsdom check in here, but perhaps the check for node can not rely on just checking window.

Reply:
Updated the function